### ISSUE TITLE
Commandline parsing to CSV?

### ISSUE THREAD

**Issue by TheRainbowConnection** [Dijji/XstReader#27]
Any chance there could be be commandline options to export to a CSV?

thanks,

**Comment by Dijji** [comment]
Yes, it is on the work list. The intent is to create a single command line tool that will support all the current export options, plus the attachment exporter created by stephenjannin in #24

No date has yet been planned, however.

Dijji

**Comment by Dijji** [comment]
This capability has now been released as part of Version 1.14. See the updated README on the front page for details.
